Dockerfile

```diff
@@ -27,5 +27,7 @@ RUN pip3 install --no-cache-dir -r requirements.txt && \
 # Copy jobs into deploy directory so nomad clients can run them
 COPY fim_mosaicker /deploy/fim_mosaicker
```

Comment:
We need to bring changes from main into this branch. Dockerfiles have split now.